src/ui/RealmBrowser/Content/index.tsx

```diff
@@ -471,10 +481,22 @@ class ContentContainer extends React.Component<
         property &&
         property.type === 'object' &&
         property.objectType &&
-        cellValue &&
+        (cellValue || property.isEmbedded) &&
         this.props.onClassFocussed
       ) {
-        this.props.onClassFocussed(property.objectType, cellValue);
+        // Handle Embedded Objects
+        if (property.isEmbedded) {
+          if (cellValue === null && property.name && property.objectType) {
+            this.onShowCreateObjectDialog(property.objectType, {
+              key: property.name,
+              parent: rowObject,
+            });
+          } else if (this.props.onListFocussed) {
+            this.props.onListFocussed(rowObject, property, cellValue);
```

Comment:
I think we should create a different focus `kind` for single objects since with this, a single object will be come "draggable", the sidebar will show "List of {className}` and the first column will be an index into the list:

<img width="818" alt="embedded-object" src="https://user-images.githubusercontent.com/1243959/91956094-706c0180-ed04-11ea-8eae-a7bd85a46fb9.png">

Reply:
The issues mentioned here are fixed in commit https://github.com/realm/realm-studio/pull/1321/commits/f25aee1cb4352518b0df384e766dc79ec64288ef, though without introducing a new focus.
The index-column was added in `derivePropertiesFromProperty`, so either way it wouldn't have been enough with a new focus type.